src/test/java/seedu/address/logic/commands/ExportCommandTest.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.logic.commands.ExportCommand.MESSAGE_EMAIL_SUCCESS;
+import static seedu.address.logic.commands.ExportCommand.MESSAGE_PHONE_SUCCESS;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
+public class ExportCommandTest {
```

Comment:
Good test cases to check the feedback message.
Can consider checking whether the output file contains the correct exported information (email, phone).

Reply:
Yep. Will do thanks